library/src/main/java/com/codetroopers/betterpickers/timezonepicker/TimeZoneInfo.java

```diff
@@ -218,6 +223,14 @@ public synchronized CharSequence getGmtDisplayName(Context context) {
         return transitions;
     }
 
+    public static long[] copyFromIntArray(int[] source) {
```

Comment:
I assumed it could not be null, but it's probably a good idea to check.
I'll fix that